data/kg/compound_protein_go_term_1/meta.yaml

```diff
@@ -90,5 +90,5 @@ bibtex:
       \ Research},\nvolume = {49},\nnumber = {16},\npages = {e96-e96},\nyear = {2021},\nmonth = {06},\nissn = {0305-1048},\ndoi = {10.1093/nar/gkab543},\n\
       url = {https://doi.org/10.1093/nar/gkab543},\n}"
 templates:
-    - The {node1_type#} {SMILES#} {rel1_type#} the {node2_type#} {node2_protein_names#} which {rel2_type#} the {node3_name#}.
-    - The {node1_type#} {SMILES#} {rel1_type#} the {node2_type#} {node2_protein_names#}. The {node2_type#} {node2_protein_names#} {rel2_type#} the {node3_name#}.
+    - The {node1_type#} with {SMILES__description} {SMILES#} {rel1_type#} the {node2_type#} {node2_protein_names#} which {rel2_type#} the {node3_name#}.
+    - The {node1_type#} with {SMILES__description} {SMILES#} {rel1_type#} the {node2_type#} {node2_protein_names#}. The {node2_type#} {node2_protein_names#} {rel2_type#} the {node3_name#}.
```

Comment:
```suggestion
    - The {node1_type#} {SMILES#} {rel1_type#} the {node2_type#} {node2_protein_names#}. The {node2_type#} {node2_protein_names#} {rel2_type#} the {node3_name#}.
```

Reply:
why would you remove this? I remember `SMILES_description` being something like "SMILES" or "InChI" and I feel something like 

"molecule with SMILES ... has ..." more natural than "molecule SMILES ... has ..."

Reply:
I don't want to test all those templates again, so I used the old template.